common/app/services/S3.scala

```diff
@@ -147,12 +147,13 @@ object S3FrontsApi extends S3 {
   lazy val stage = Configuration.facia.stage.toUpperCase
   val namespace = "frontsapi"
   lazy val location = s"$stage/$namespace"
+  private val filename = "pressed.v2.json"
```

Comment:
manually :)
In the past`pressed.json` files have been moved from the root of a front directory to the `fapi` subdirectory and the old ones were never deleted. I always wanted to clean up those. Now we will have to clean both `pressed.json` ✂️